### PR TITLE
Added a length limit to denominations.

### PR DESCRIPTION
**Pull request by aletheist**
Thanks to @Ganon11  I was able to test and the patch is confirmed to work.

Issue #27

### CODE REVIEW

File: denom.py [hunk 1]
```diff
@@ -3,12 +3,17 @@
 @sopel.module.commands('setdenom', 'mydenom')
 @sopel.module.example('.setdenom Lutheran')
 def set_denom(bot, trigger):
-  '''Check whether something (or someone) is honorable'''
+  '''Set a user's denomination'''
+  length_limit=128
   person = str(trigger.nick)
-  denom = 'Goof'
+  denom = 'Trout'
```
Comment (Ganon11) [inline denom.py]: Changing the default denomination is serious business. Don't know if I want to be changing it willy-nilly.

Plus, Trout are heretics.

File: denom.py [hunk 2]
```diff
@@ -3,12 +3,17 @@
 @sopel.module.commands('setdenom', 'mydenom')
 @sopel.module.example('.setdenom Lutheran')
 def set_denom(bot, trigger):
-  '''Check whether something (or someone) is honorable'''
+  '''Set a user's denomination'''
```
Comment (Ganon11) [inline denom.py]: Oops.

File: denom.py [hunk 3]
```diff
@@ -3,12 +3,17 @@
...
+  length_limit=128
   person = str(trigger.nick)
-  denom = 'Goof'
+  denom = 'Trout'
   if trigger.group(2):
     denom = trigger.group(2)
 
+  if len(denom) > length_limit:
+    bot.reply('Denomination name too long. (Limit {0} characters)'.format(str(length_limit)))
```
Comment (Ganon11) [inline denom.py]: Is .format better than the %-style string-replace? If so, can you change the bot.reply lines below? If not, let's stick with just one style for consistency.

File: denom.py [hunk 4]
```diff
@@ -3,12 +3,17 @@
 @sopel.module.commands('setdenom', 'mydenom')
 @sopel.module.example('.setdenom Lutheran')
 def set_denom(bot, trigger):
-  '''Check whether something (or someone) is honorable'''
+  '''Set a user's denomination'''
+  length_limit=128
```
Comment (Ganon11) [inline denom.py]: length_limit = 128

with the extra spaces. Style!

File: denom.py (hunk 1 above)
Comment (aletheist) [inline denom.py]: Of course trouts are heretics. That's the point.

**Pull request merged**